tardis/montecarlo/struct.py

```diff
@@ -0,0 +1,190 @@
+import os
+from ctypes import *
+
+import numpy as np
+from tardis import __path__ as path
+
+test_path = os.path.join(path[0], 'montecarlo', 'test_montecarlo.so')
+tests = CDLL(test_path)
+
+cmontecarlo_filepath = os.path.join(path[0], 'montecarlo', 'montecarlo.so')
+cmontecarlo_methods = CDLL(cmontecarlo_filepath)
```

Comment:
The loading of the CDLL is something specific to the tests, this shouldn't be in struct.py, only the definition of `RPacket` and `StorageModel`